src/main/java/de/tum/in/www1/artemis/repository/SubmissionRepository.java

```diff
@@ -111,15 +111,12 @@
      *         exercise due date at all
      */
     @Query("""
-            SELECT COUNT (DISTINCT p) FROM StudentParticipation p
+            SELECT COUNT (DISTINCT p) FROM StudentParticipation p join  p.submissions s
```

Comment:
Code looks good, but I find it a little bit strange that you are counting the participations here and not the submission, but I guess it is fine if it works :) Gj

Reply:
Its legacy code, I only rewrote this part of the query to include performance improvements 😄. We will refactor some of these old queries in a followup.